aws-s3/s3.mli

```diff
@@ -38,7 +38,11 @@ module Make(Io : Types.Io) : sig
   }
 
   type nonrec 'a result = ('a, error) result Deferred.t
-  type 'a command = ?credentials:Credentials.t -> ?connect_timeout_ms:int -> endpoint:Region.endpoint -> 'a
+
+  (* If [confirm_requester_pays], caller acknowledges that it will pay
```

Comment:
Thanks for placing a comment here, but It will not be visible in the documentation. What does the documentation look like if this is changed to be a documentation comment (i.e. starting with `(**`)